packages/esm-ohri-pmtct-app/src/api/api.ts

```diff
@@ -82,6 +82,17 @@ export function getEstimatedDeliveryDate(patientUuid: string, pTrackerId: string
   });
 }
 
+export function getIdentifierInfo(identifier: string){
```

Comment:
@kajambiya this function looks solid to me but the only thing missing here is HTTP request & response error handling, would be good if you can add to this function and we show this to the other devs otherwise we will keep having those errors that emerge out of the blue and the user doesn't understand. Here in case an HTTP request or response error happens I would expect to see a message like **Failed to retrieve identifier information**. Also shouldn't this function be marked as `async` ?